src/status_im/ui/screens/chat/utils.cljs

```diff
@@ -13,9 +13,9 @@
 
 (defn format-author [alias style name]
   (let [additional-styles (style false)]
-    (if name
-      [react/text {:number-of-lines 1
-                   :style (merge {:color       colors/blue
+    (if (and name
+             (< (count name) 40))
```

Comment:
@yenda  here is a full name with domain, so it can be like `myname.domain1.domain2.eth` also we allow to refister 40 symbols for status name, but here we don't count stateofus.eth domain length

Reply:
so hiding names not a solution for sure, because we should show such names for sure `myname.domain1.domain2.eth`